pkg/model/base.go

```diff
@@ -0,0 +1,156 @@
+package model
```

Comment:
Please deprecate use of 'model' ... we can simply marshall needed information in main.go and pass it to the nsmgr.NewServer(...) rather than bringing all of this complexity.